services/galley/src/Galley/API/Public/Servant.hs

```diff
@@ -141,8 +142,8 @@ servantSitemap =
               setTeamSearchVisibilityAvailableInternal
               . DoAuth
           )
-        <@> mkNamedAPI @"get-search-visibility" getSearchVisibility
-        <@> mkNamedAPI @"set-search-visibility" setSearchVisibility
+        <@> mkNamedAPI @"get-search-visibility" (getSearchVisibility . tUnqualified)
```

Comment:
Please change the type of `getSearchVisibility` to take a `Local UserId` instead of a `UserId` now that you used `ZLocalUser` instead of `ZUser`.